core/src/main/scala/dagr/core/tasksystem/ProcessTask.scala

```diff
@@ -57,12 +57,15 @@ trait ProcessTask extends UnitTask {
     */
   def args: Seq[Any]
 
+  protected var quoteIfNecessary: Boolean = true
```

Comment:
I am completely open how to implement this (see `ShellCommandAsIs`) but I really do want the functionality to skip the `ProcessTask.quoteIfNecessary` application.